src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -303,7 +293,7 @@ export const DatasetSearchTable = (props) => {
         </Box>
         <Box sx={{display: 'flex', flexDirection: 'row', paddingTop: '2em'}}>
           <Box sx={{width: '14%', padding: '0 1em'}}>
-            <DatasetFilterList datasets={datasets} filterHandler={filterHandler} isFiltered={isFilteredArray} onClear={() => setFilters(defaultFilters)}/>
+            <DatasetFilterList datasets={datasets} filterHandler={filterHandler} filters={filters} isFiltered={isFilteredArray} onClear={() => setFilters(defaultFilters(datasets))}/>
```

Comment:
Something about this is breaking the `dataset_search_table.spec.js` tests, but it's not exactly clear what. I think the tests might be a little brittle as well in that they're expecting a populated `response.response.body[0]` object.

Reply:
I think I see it. The search filter text is slightly different in this PR which breaks the test. Changing line 68 in `dataset_search_table.spec.js` to:
```
return handler(req, '{"range":{"participantCount":{"gte":100,"lte":"50"}}}');
```
and line 91 to
```
return handler(req, '{"range":{"participantCount":{"gte":100,"lte":100}}}');
```
allows the string matching in the handler function to work correctly.